test/controllers/thesis_controller_test.rb

```diff
@@ -231,19 +231,7 @@ def attach_files_to_records(tr, th)
     assert_response :success
   end
 
-  test 'processing queue shows nothing without work done' do
```

Comment:
Even though it's highly unlikely we'll ever see this state, I'm a bit reluctant to remove this test. Maybe we could add something like `Thesis.all.map { |t| t.files.delete_all }` to remove all the file fixtures in the scope of the test?